nodejs/dist/package.json

```diff
@@ -2,10 +2,14 @@
   "name": "uws",
   "version": "0.3.1",
   "scripts": {
-    "build": ""
+    "preinstall": "node-gyp rebuild && exit 0; echo 'Falling back to precompiled binaries';"
```

Comment:
Yes, npm controls the env and actually somewhat parses the command here.
For example i couldn't do `node-gyp rebuild || echo "xxx"`